src/engine/controls/cuecontrol.cpp

```diff
@@ -124,169 +126,9 @@ CueControl::CueControl(const QString& group,
-    connect(m_pIntroEndActivate, &ControlObject::valueChanged,
-            this, &CueControl::introEndActivate,
-            Qt::DirectConnection);
-
-    m_pOutroStartPosition = new ControlObject(ConfigKey(group, "outro_start_position"));
-    m_pOutroStartPosition->set(Cue::kNoPosition);
-
-    m_pOutroStartEnabled = new ControlObject(ConfigKey(group, "outro_start_enabled"));
-    m_pOutroStartEnabled->setReadOnly();
-
-    m_pOutroStartSet = new ControlPushButton(ConfigKey(group, "outro_start_set"));
-    connect(m_pOutroStartSet, &ControlObject::valueChanged,
-            this, &CueControl::outroStartSet,
-            Qt::DirectConnection);
-
-    m_pOutroStartClear = new ControlPushButton(ConfigKey(group, "outro_start_clear"));
-    connect(m_pOutroStartClear, &ControlObject::valueChanged,
-            this, &CueControl::outroStartClear,
-            Qt::DirectConnection);
-
-    m_pOutroStartActivate = new ControlPushButton(ConfigKey(group, "outro_start_activate"));
-    connect(m_pOutroStartActivate, &ControlObject::valueChanged,
-            this, &CueControl::outroStartActivate,
-            Qt::DirectConnection);
-
-    m_pOutroEndPosition = new ControlObject(ConfigKey(group, "outro_end_position"));
-    m_pOutroEndPosition->set(Cue::kNoPosition);
-
-    m_pOutroEndEnabled = new ControlObject(ConfigKey(group, "outro_end_enabled"));
-    m_pOutroEndEnabled->setReadOnly();
-
-    m_pOutroEndSet = new ControlPushButton(ConfigKey(group, "outro_end_set"));
-    connect(m_pOutroEndSet, &ControlObject::valueChanged,
-            this, &CueControl::outroEndSet,
-            Qt::DirectConnection);
-
-    m_pOutroEndClear = new ControlPushButton(ConfigKey(group, "outro_end_clear"));
-    connect(m_pOutroEndClear, &ControlObject::valueChanged,
-            this, &CueControl::outroEndClear,
-            Qt::DirectConnection);
-
-    m_pOutroEndActivate = new ControlPushButton(ConfigKey(group, "outro_end_activate"));
-    connect(m_pOutroEndActivate, &ControlObject::valueChanged,
-            this, &CueControl::outroEndActivate,
-            Qt::DirectConnection);
-
-    m_pVinylControlEnabled = new ControlProxy(group, "vinylcontrol_enabled");
-    m_pVinylControlMode = new ControlProxy(group, "vinylcontrol_mode");
-
-    m_pHotcueFocus = new ControlObject(ConfigKey(group, "hotcue_focus"));
-    setHotcueFocusIndex(Cue::kNoHotCue);
-
-    m_pHotcueFocusColorPrev = new ControlObject(ConfigKey(group, "hotcue_focus_color_prev"));
-    connect(m_pHotcueFocusColorPrev,
-            &ControlObject::valueChanged,
-            this,
-            &CueControl::hotcueFocusColorPrev,
-            Qt::DirectConnection);
-
-    m_pHotcueFocusColorNext = new ControlObject(ConfigKey(group, "hotcue_focus_color_next"));
-    connect(m_pHotcueFocusColorNext,
-            &ControlObject::valueChanged,
-            this,
-            &CueControl::hotcueFocusColorNext,
+    m_pPassthrough = make_parented<ControlProxy>(group, "passthrough");
```

Comment:
Here the parent is missing

Reply:
fixed
though I wonder how `make_parented` doesn't fail without a parent on creation, only on destruction.